reference/api/experimental_features.mdx

```diff
@@ -19,15 +19,15 @@ The experimental API route is not compatible with all experimental features. Con
 {
   "metrics": false,
   "logsRoute": true,
-  "vectorStore": false,
+  "containsFilter": false,
+  "editDocumentsByFunction": false
```

Comment:
might need to be changed in another PR, but this now contains `"network": false` as well

Reply:
yeap, I'm adding `network` in another PR, which should be ready for review later today.